composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "require-dev": {
+        "phpunit/phpunit": "^9.6"
+    },
+    "require": {
+        "silverstripe/supported-modules": "dev-pulls/main/combine-all-versions"
+    },
+    "repositories": {
+        "silverstripe/supported-modules": {
+            "type": "vcs",
+            "url": "git@github.com:creative-commoners/supported-modules.git"
+        }
+    }
```

Comment:
```suggestion
```

Reply:
Discussion in https://github.com/silverstripe/gha-generate-matrix/pull/87#discussion_r1590480227

Reply:
Done